### PR TITLE
SUS-182: check for null title

### PR DESCRIPTION
**Pull request by owend**
Stealin tickets :shipit: @mixth-sense 

@Wikia/sustaining-team

### CODE REVIEW

File: extensions/wikia/VideoHandlers/VideoFileUploader.class.php [hunk 1]
```diff
@@ -112,6 +112,10 @@ public function upload( &$oTitle ) {
 		}
 		$oTitle = Title::newFromText( $this->getNormalizedDestinationTitle(), NS_FILE );
 
+		if ( $oTitle == null ) {
```
Comment (inez) [inline extensions/wikia/VideoHandlers/VideoFileUploader.class.php]: How about `===`?

Comment (ghost) [inline extensions/wikia/VideoHandlers/VideoFileUploader.class.php]: I'd go with `! $oTitle instanceof Title`.

Comment (inez) [inline extensions/wikia/VideoHandlers/VideoFileUploader.class.php]: I don't see a need for that, it seem to return null or a Title object only.

Comment (ghost) [inline extensions/wikia/VideoHandlers/VideoFileUploader.class.php]: :+1:

**Comment by ukaka** [comment]
LGTM. Passed QA.

**Pull request merged**